src/features/donations/screens/ThankYou.tsx

```diff
@@ -182,7 +182,17 @@ function ThankYou({
                 />
               </div>
               <div className={styles.donationCount}>
-                {t('donate:myTreesPlantedByOnLocation', {
+                {donation.treeCount == 1 ?
```

Comment:
Please let's use the technology already existing in the i18n module automatically using a plural resource if the given parameter `count` does not equals `1` as e.g. already used at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/498672c39de909e83f2445f8f47dc988aa794beb/src/features/donations/screens/ContactDetails.tsx#L87-L93 plus the related resources at https://github.com/Plant-for-the-Planet-org/planet-webapp/blob/498672c39de909e83f2445f8f47dc988aa794beb/public/static/locales/en/donate.json#L2-L3
The parameter `count` is only used to decide which resource (singular or plural) to choose while the formatted parameter `treeCount` is actually used to be displayed.